ci_framework/plugins/modules/generate_make_tasks.py

```diff
@@ -59,13 +59,21 @@
 
 
 MAKE_TMPL = '''---
-- name: Run `make %(target)s`
+- name: Debug make_%(target)s_params
+  when: make_%(target)s_params is defined
+  ansible.builtin.debug:
+    var: make_%(target)s_params
+- name: Debug make_%(target)s_environment
+  ansible.builtin.debug:
+    msg: "{{ edpm_env | combine(make_%(target)s_env|default({})) }}"
+- name: Run %(target)s
   ci_make:
-    output_directory: "{{ cifmw_install_yamls_basedir }}/artifacts"
+    output_dir: "{{ cifmw_basedir }}/artifacts"
     chdir: "%(chdir)s"
     target: %(target)s
-    dry_run: "{{ cifmw_install_yamls_dryrun|bool }}"
-  environment: "{{ edpm_env }}"
+    dry_run: "{{ make_%(target)s_dryrun|default(false)|bool }}"
+    params: "{{ make_%(target)s_params|default(omit) }}"
+  environment: "{{ edpm_env | combine(make_%(target)s_env|default({})) }}"
```

Comment:
fstring is supported by default after python 3.5. Can we use the same here like this.
```
MAKE_TMPL = '''---
- name: Run `make %(target)s`
- name: Debug make_%(target)s_params
  when: make_%(target)s_params is defined
  ansible.builtin.debug:
    var: make_%(target)s_params
- name: Debug make_%(target)s_environment
  ansible.builtin.debug:
    msg: "{{ edpm_env | combine(make_%(target)s_env|default({})) }}"
- name: Run %(target)s
  ci_make:
    output_directory: "{{ cifmw_install_yamls_basedir }}/artifacts"
    output_dir: "{{ cifmw_basedir }}/artifacts"
    chdir: "%(chdir)s"
    target: %(target)s
    dry_run: "{{ cifmw_install_yamls_dryrun|bool }}"
  environment: "{{ edpm_env }}"
    dry_run: "{{ make_%(target)s_dryrun|default(false)|bool }}"
    params: "{{ make_%(target)s_params|default(omit) }}"
  environment: "{{ edpm_env | combine(make_%(target)s_env|default({})) }}"
'''
```

Reply:
I'd rather avoid - the `{{{{` is really not nice imho.